server/bleep/src/webserver/answer/exchange.rs

```diff
@@ -183,28 +154,28 @@ pub enum SearchStep {
 #[derive(Debug)]
 pub enum Update {
     Step(SearchStep),
-    Filesystem(Vec<SearchResult>),
+    Filesystem(Vec<FileAction>),
     Article(String),
-    Finalize(String),
+    Conclude(String),
 }
 
 #[derive(serde::Serialize, serde::Deserialize, Debug, Clone)]
-pub enum SearchResult {
-    Cite(CiteResult),
-    Directory(DirectoryResult),
-    Modify(ModifyResult),
-    Conclude(ConcludeResult),
+pub enum FileAction {
```

Comment:
Not sure about the renaming here. Feel like this overlaps too much with the notion of Agent`Action`s. These don't really _do_ anything, so I prefer the `Result` terminology.